libvast/src/format/arrow.cpp

```diff
@@ -19,6 +19,7 @@
 #include "vast/table_slice_builder.hpp"
 #include "vast/type.hpp"
 
+#include <arrow/api.h>
 #include <arrow/util/config.h>
```

Comment:
Is this include still required with the removed #ifdef below?

Reply:
My mildly insufficient understanding of C++ makes me think that without that import, C++ only sees a forward declaration of the `RecordBatch`, which is fixed by importing this header. Maybe there's a better solution?

```
/home/pi/src/github/tenzir/vast/libvast/src/format/arrow.cpp:52:28: error: invalid use of incomplete type ‘using element_type = class arrow::RecordBatch’ {aka ‘class arrow::RecordBatch’}
```

Reply:
The comment is about the `arrow/util/config.h`, not the `arrow/api.h` include. Your understanding for the latter one is correct, though.

Reply:
gotcha, fixed.